Configuration/Eras/python/Era_Run2_2017_cff.py

```diff
@@ -9,6 +9,7 @@
 from Configuration.Eras.Modifier_run2_HEPlan1_2017_cff import run2_HEPlan1_2017
 from Configuration.Eras.Modifier_trackingPhase1_cff import trackingPhase1
 from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
+from Configuration.Eras.Modifier_run2_CSC_2017_cff import run2_CSC_2017
 
-Run2_2017 = cms.ModifierChain(Run2_2016, phase1Pixel, run2_ECAL_2017, run2_HF_2017, run2_HCAL_2017, run2_HE_2017, run2_HEPlan1_2017, trackingPhase1, run2_GEM_2017)
+Run2_2017 = cms.ModifierChain(Run2_2016, phase1Pixel, run2_ECAL_2017, run2_HF_2017, run2_HCAL_2017, run2_HE_2017, run2_HEPlan1_2017, trackingPhase1, run2_GEM_2017, run2_CSC_2017)
```

Comment:
hi @dildick - you expect that run2_CSC_2017 will also be appropriate for 2017 data taken so far?